packages/manager/apps/hub/src/dashboard/new-pricing/template.html

```diff
@@ -0,0 +1,189 @@
+<div class="mb-3">
+    <h2 data-translate="manager_hub_new_pricing_title"></h2>
+    <p>
+        <span data-translate="manager_hub_new_pricing_description1"></span
+        ><br />
+        <span data-translate="manager_hub_new_pricing_description2"></span>
+    </p>
+    <p data-translate="manager_hub_new_pricing_explain"></p>
+    <ul>
+        <li data-translate="manager_hub_new_pricing_explain1"></li>
+        <li data-translate="manager_hub_new_pricing_explain2"></li>
+    </ul>
+</div>
+
+<oui-datagrid data-rows="$ctrl.getRepricing">
```

Comment:
**nitpick:** `getRepricing` sounds like a function. Can you please rename the variable?

Reply:
@anooparveti  updated. Thanks